### PR TITLE
Split big JSON assert method into multiple small ones

### PR DESCRIPTION
**Pull request by dstepanov**

### CODE REVIEW

File: accurest-core/src/main/groovy/io/codearte/accurest/builder/BlockBuilder.groovy [hunk 1]
```diff
@@ -72,6 +78,13 @@ class BlockBuilder {
 		return this
 	}
 
+	String spacer() {
+		spacer
```
Comment (marcingrzejszczak) [inline accurest-core/src/main/groovy/io/codearte/accurest/builder/BlockBuilder.groovy]: returns are missing

**Comment by marcingrzejszczak** [comment]
I'd wait with this merge until messaging is finished

**Comment by dstepanov** [comment]
Fixed returns.
WDYT about implementation?

**Comment by marcingrzejszczak** [comment]
Maybe it would be better to match the files?

```
given:
// a big JSON read from a file
then:
// assert that output from a file is ok?
```

I'm just brainstorming

**Comment by dstepanov** [comment]
Where?
1. 4. 2016 v 19:39, Marcin Grzejszczak notifications@github.com:

**Comment by marcingrzejszczak** [comment]
You're splitting a big file into methods to assert parts of it.What if those assertions were in a file that we would load at runtime. It could be a list of JSON Paths for example

**Comment by dstepanov** [comment]
That can work too, but do we have stuff like optional etc. in stub request body?

**Comment by marcingrzejszczak** [comment]
Yes you can have optional for a part of the request, for the stub side. 

BTW if you come with another idea let's first discuss it ok? We can have a chat here - https://gitter.im/Codearte/accurest

**Comment by marcingrzejszczak** [comment]
Let's close it for now

**Pull request closed without merging**